core/src/main/java/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor.java

```diff
@@ -112,6 +113,7 @@ public HttpResponse doIndex() throws IOException {
      */
     // used by Jelly view
     @Restricted(NoExternalUse.class)
+    @SuppressFBWarnings("NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE")
     public boolean hasFreshToken(@Nonnull User user, @Nullable ApiTokenProperty.TokenInfoAndStats legacyStats) {
```

Comment:
Maybe just `CheckForNull`?

Reply:
> Maybe just `CheckForNull`?

I'll try that and see how it goes in CI.

Reply:
> I'll try that and see how it goes in CI.

I tried this suggestion, but [it didn't work](https://ci.jenkins.io/job/Core/job/jenkins/job/PR-4062/3/consoleFull). The odd thing, though, is that neither did [my original attempt](https://ci.jenkins.io/job/Core/job/jenkins/job/PR-4062/2/consoleFull) using `@SuppressFBWarnings("NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE")`. So I'm at a loss as to how to fix this particular warning at the moment. Part of the problem is that I can't seem to reproduce the error locally by running `mvn install`. If I could do that, then I might be able to debug the cause more quickly than by using Jenkins' CI system.